public/styles.scss

```diff
@@ -63,6 +64,9 @@ $darkGrey: #434651;
   position: fixed;
   background: $backgroundColor;
   transition: left 0.3s ease-out;
+  border-top: 1px solid $lightborderColor;
+  border-bottom: 1px solid $lightborderColor;
+
```

Comment:
This almost works but the bottom border isn't visible. ( due to the outline being outside the viewport )
![image](https://github.com/TrevorBrage/TradingviewPlus/assets/55461688/dd50487c-9d6c-4371-8a52-ea0ce1ceab9f)

Adding `box-sizing: border-box` should fix this.